src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 800;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, serif;
+}
+
+.search__bar {
+  position: relative;
+  margin-top: 20px;
+  box-sizing: border-box;
+}
+
+.search__input {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  width: 100%;
+  flex: 1;
+  outline: none;
+}
+
+.big__input {
+  line-height: 68px;
+  padding: 0 0 0 62px;
+}
+
+.big__input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.small__input {
+  line-height: 40px;
+  padding: 0 0 0 33px;
+}
+
+.small__input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
```

Comment:
Don't Repeat Yourself (DRY)
You will have styles for `.search-bar__input` and for its states (`:hover/:focus`)

 So when you use modifier `search-bar__input--small` just add styles that make difference. Do not repeat those styles that you have already pass to block element `.search-bar__input`
 
 Check and apply all over the document